src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,7 @@
+package racingcar.domain
+
+data class Car(val carName: String, private val movingDistance: Int) {
+    fun movingDistance(): Int {
+        return movingDistance
+    }
```

Comment:
아래와 같이 작성해볼 수도 있겠네요!

```suggestion
data class Car(val carName: String, val movingDistance: Int)
```

다만 `Car` 객체를 외부에서 생성해서 리턴해주지 말고 `Car` 객체에 일부 역할과 책임을 위임할 수도 있지 않을까요? (이어서)

Reply:
private를 제거하고 Car 객체안에서 데이터를 만들도록 수정해보았습니다